cartographer_ros/cartographer_ros/sensor_bridge.cc

```diff
@@ -147,30 +147,29 @@ void SensorBridge::HandleLaserScan(
         points.points.size() * i / num_subdivisions_per_laser_scan_;
```

Comment:
Add a TODO that subdivisions should go away and instead the per-point timing be used in Cartographer.